src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -83,6 +87,10 @@ public CursorsController(final CursorsService cursorsService,
                                            @NotNull @RequestHeader("X-Nakadi-StreamId") final String streamId,
                                            final NativeWebRequest request,
                                            final Client client) {
+
+        LOG.debug("[COMMIT_CURSORS] committing {} cursor(s) for subscription {}", cursors.getItems().size(),
```

Comment:
This controller does not only commits so I would not like to include that to logger itself. And anyway these logs are temporary, they will be removed after the problem with performance is resolved.